shared-libs/cht-datasource/test/local/person.spec.ts

```diff
@@ -228,6 +228,7 @@ describe('local person', () => {
     describe('getPage', () => {
```

Comment:
Do we need to add some test cases for throwing an exception when there is an invalid cursor value?  (The new logic you added in `local/person.ts`).

Reply:
ahh. yes. good catch.